src/style.css

```diff
@@ -0,0 +1,136 @@
+@import "./variables.css";
+
+body {
+  margin: 0;
+  font-family: "Roboto", Helvetica, sans-serif;
+  font-style: normal;
+  /* #region variables */
+  --card-width: 200px;
+  --black-color: #060b35;
+  --grey-color: #616070;
+  --white-color: #fff;
+  --blue-color: #00acdc;
+  --border-color-main: #f3f3f3;
+  /* #endregion */
+}
+
+.product {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
```

Comment:
move `box-sizing` to `*` selector

Reply:
If I move the box-sizing to * selector then the test fails (image shrinks).

Reply:
this means that you have set the dimensions minus the borders and paddings, you should check all the dimensions again